### DISCUSSION TITLE
Why does GodMode9 have a "Brick my 3DS" option when pressing home while holding down R?

### DISCUSSION THREAD

**Discussion by IAmTheNerdNextDoor** [d0k3/GodMode9#801] [Q&A] [answered]
I don't know why it exists.

**Comment by d0k3** [comment, marked as answer]
Just try it. I assure you it will not, actually brick your 3DS.

**Reply by IAmTheNerdNextDoor** [reply]
Are you sure? I really do not want to brick my system. Are there any videos of it?

Edit: saw a gbatemp post https://gbatemp.net/threads/godmode9s-secret-option-paint9.585241/ I'm gonna try it

**Reply by IAmTheNerdNextDoor** [reply]
damn this is pretty fun
